src/components/inputs/localized-rich-text-input/languages-control.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import FlatButton from '../../buttons/flat-button';
+import { WorldIcon } from '../../icons';
+import messages from './messages';
+
+const LanguagesControl = props => {
+  const intl = useIntl();
+  return (
+    <FlatButton
+      icon={<WorldIcon />}
+      label={intl.formatMessage(
+        props.isClosed ? messages.show : messages.hide,
```

Comment:
basically copy pasted from `LocalizedMultilineTextInput` and `LocalizedMoneyInput`